src/Microsoft.OData.Edm/Csdl/Serialization/EdmModelCsdlSerializationVisitor.cs

```diff
@@ -344,7 +344,16 @@ protected override void ProcessFunctionImport(IEdmFunctionImport functionImport)
 
         protected override void ProcessAnnotation(IEdmVocabularyAnnotation annotation)
         {
-            bool isInline = IsInlineExpression(annotation.Value);
+            bool isInline;
+            // Note: This currently needs to be separated out because CsdlSemanticsVocabularyAnnotation
+            // doesn't return the default value from Value at the moment.

```

Comment:
Right. This should go away if we can fix the code that returns the default value in annotation.Value.

Reply:
Done! ✅